docs/build.md

```
@@ -38,15 +34,46 @@ Use this when you want a build to self-update from your own fork releases instea
 node scripts/pack.js --update-repo owner/repo --version nightly-$(git rev-parse --short HEAD) --output tmp/app.asar
 ```
 
+## Build With A Custom Update Channel
```

markdownlint-cli2 (0.23.2)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)